docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -424,299 +442,320 @@ In Geo-Partitioning, the table must be splitted into partitions, and each partit
-
-Expression indices and invisible indices are also supported, as the key format is the same as normal.
-
-Defining placement of an unpartitioned index is in such a statement:
+In this example, partition `p0` uses the policy `storeonhdd`, partition `p4` uses the policy `storeonfastssd` and the remaining partitions use the policy `companystandardpolicy`. Assuming the following `ALTER TABLE` statement is executed, only partitions `p1`-`p3` will be updated:
 
 ```sql
-ALTER TABLE table_name ALTER INDEX index_name
-	{ADD | ALTER} PLACEMENT POLICY CONSTRAINTS=constraints ROLE=role REPLICAS=replicas,
-	...
-	
-ALTER TABLE table_name ALTER INDEX index_name
-	DROP PLACEMENT POLICY ROLE=role,
-	...
+ALTER TABLE t1 PLACEMENT POLICY=`companynewpolicy`;
 ```
 
-This key format of an unpartitioned index is `t_{table_id}_i_{index_id}_r_{pk_value}`. The key range can be inferred by `table_id` and `index_id`.
-
-### Partitioned index placement
-
-Defining placement rules of an index in one specific partition is in such a statement:
-
+For the syntax:
 ```sql
-ALTER TABLE table_name ALTER PARTITION partition_name ALTER INDEX index_name
-	{ADD | ALTER} PLACEMENT POLICY CONSTRAINTS=constraints ROLE=role REPLICAS=replicas,
-	...
-	
-ALTER TABLE table_name ALTER PARTITION partition_name ALTER INDEX index_name
-	DROP PLACEMENT POLICY ROLE=role,
-	...
-```
+ALTER TABLE pt
+    EXCHANGE PARTITION p
+    WITH TABLE nt;
+```	
 
-The key format of partitioned index is `t_{partition_id}_i_{index_id}_r_{pk_value}`. The key range can be inferred by `partition_id` and `index_id`.
+If `nt` has placement rules associated with it, they will be retained when it becomes a partition of table `pt`. However, if no placement rules have been specified, then the rules of the table `pt` will be used. This helps protect against the case that a partition may need to have "default" placement rules, but default does not mean what the table uses (the output of `SHOW CREATE TABLE` would appear ambiguous). When the partition `p` is converted to table `nt`, it will continue to use the rules it had as a partition (either explicitly listed for the partition or the default for the table).
 
-When an index is partitioned, defining placement rule of the whole index at once is not supported. It will involve multiple key ranges, and the scenario of its application is rare.
+This behavior is inspired by how a `CHARACTER SET` or `COLLATE` attribute applies to a column of a table, and columns will use the character set defined at the table-level by default.
 
-For example, `t` is a partitioned table and `idx` is the index on `t`. It’s not supported to do this:
+#### Removing placement from a database, table or partition
+
+Placement policy can be removed from an object via the following syntax:
 
 ```sql
-ALTER TABLE `t` ALTER INDEX `idx`
-	ADD PLACEMENT POLICY ...
+ALTER DATABASE test [DEFAULT] PLACEMENT SET DEFAULT;
+ALTER TABLE t1 PLACEMENT=default;
```

Comment:
Should it be `PLACEMENT POLICY`?

Reply:
Yes. Previously this would have made sense, but it should be `PLACEMENT POLICY` now that the direct placement options are all considered `table_options`.

Fixed in https://github.com/pingcap/tidb/pull/26221/commits/ac3d02d9473ce638fda81caac74a5b2f8c38f97b